imap_processing/tests/swe/test_swe_l2.py

```diff
@@ -60,10 +66,39 @@ def test_calculate_phase_space_density(patch_get_particle_energy):
     # Using this in the formula, we calculate expected density value.
     expected_calculated_density = (2 * 1) / (1 * VELOCITY_CONVERSION_FACTOR * 1**2)
     expected_density = np.full((24, 30, 7), expected_calculated_density)
-    assert np.all(density[0].data == expected_density)
+    assert np.all(
+        phase_space_density_ds["phase_space_density"][0].data == expected_density
+    )
```

Comment:
Is there a reason you are not using `np.testing.assert_array_equal` here? I think that is preferred to this implementation.